nixos/modules/programs/qdmr.nix

```diff
@@ -0,0 +1,27 @@
+{ config, lib, pkgs, ... }:
+
+{
+  meta.maintainers = [ lib.maintainers.janik ];
+
+  ###### interface
+  options = {
+    programs.qdmr = {
+      enable = lib.mkOption {
+        default = false;
+        type = lib.types.bool;
+        description = lib.mdDoc ''
+          Whether to configure system to use qdmr.
+          To grant access to a user, it must be part of dialout group:
+          `users.users.alice.extraGroups = ["dialout"];`
+        '';
+      };
+    };
+  };
+
+  ###### implementation
+  config = lib.mkIf config.programs.qdmr.enable {
+    services.udev.packages = [ pkgs.qdmr ];
```

Comment:
use `cfg.package` idiom.

Reply:
I will look at that later today or this weekend

Reply:
Pushed that earlier today :)